<launch>
  <node name="bag_player" pkg="rosbag" type="play" args="--loop $(find pwc_net)/bag/test_input.bag"/>

  <param name="action_name" value="pwc_net_action"/>

  <node name="pwc_net_action_client" pkg="pwc_net" type="action_client" required="true" output="screen">
    <remap from="image" to="stereo_robot/mobile_base/camera/left/image_raw"/>
    <param name="image_transport" value="compressed"/>
  </node>
</launch>
